samples/1_Utils/hipCommander/hipCommander.cpp

```diff
@@ -9,8 +9,6 @@
 #include <hip/hip_runtime.h>
 #ifdef __HIP_PLATFORM_HCC__
 #include <elf.h>
```

Comment:
I think this include is also not required. We should be able to remove it.

Reply:
Fixed.